### ISSUE TITLE
TypeError on Anchors

### ISSUE THREAD

**Issue by ryanwhitney** [craftcms/anchors#13]
### Description

I'm getting a TypeError which seems to be coming from the Anchors plugin. It seems to be happening inconsistently, so far just seeing it on one page.

I've removed the `|anchors` filter for now, which fixes the problem.

> TypeError
> 
> Argument 1 passed to craft\anchors\TwigExtension::anchorsFilter() must be of the type string, null given, called in 
> /Users/sitename/storage/runtime/compiled_templates/03/0309d42d1a754faba45c18319839192d4f28bf45fc11f6ad5bf6994645cbdf74.php on line 207

<img width="1041" alt="Screen Shot 2021-05-24 at 10 42 03 AM" src="https://user-images.githubusercontent.com/1018145/119387055-3d481480-bc7d-11eb-9b50-dbf58e8e8207.png">

- Craft version: Craft Pro 3.6.15
- PHP version: 7.3.26
- Database driver & version: MySQL 5.5.5
- Plugins & versions: Anchors 2.3.0

**Comment by LouisCuvelier** [comment]
Same problem here. As a workaround, you can check if your `block.text` is null before applying `anchors` like this : 
```twig
{% if block.text is not null %}
     {{ block.text|anchors('h1,h2,h3') }} 
{% endif %}
```

**Comment by brandonkelly** [comment]
Just released v2.3.1 with a fix for this.
